### ISSUE TITLE
What is the val set of pre-train

### ISSUE THREAD

**Issue by JayZhu0104** [microsoft/TAP#8]
Hi! After downloading OCR-CC features, I found that there were only feature files of training set. But I noticed that the IMDB file contains information about the val set. And the 'tap_base_pretrain.yml' file needs to fill in the val set and test set. What should be filled in this part?
Thanks a lot!
![image](https://user-images.githubusercontent.com/40751213/142958201-781c2559-e8dc-4a7b-a21c-c880b1d32b14.png)

**Comment by zyang-ur** [comment]
Hi @JayZhu0104 ,

We don't use the CC-val set and the related files are included just for completeness. 

You could either skip the validation or place the textvqa's val set in the boxed line. Since we would use the final iteration's ckpt in pretraining, it should not matter.

**Comment by JayZhu0104** [comment]
Thank you for your reply!
